### PR TITLE
feat: change colour picker

### PR DESCRIPTION
**Pull request by dan47bennett**
This is part of the work effort outlined in #21. Intent is to swap in a new colour picker.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (michaelfitzhavey) [inline package-lock.json]: This file shouldn't be here (what's it do? 😄 )

let's delete it

Comment (dan47bennett) [reply package-lock.json]: Fixed in 3183403

File: orchestration-ui/src/containers/Lights/components/Light.vue [hunk 2]
```diff
@@ -91,7 +99,7 @@ export default {
 	data() {
 		return {
 			inputVal: this.value,
-			colourPickerValue: { hex: '#ffffff' },
+			colourPickerValue: '',
```
Comment (michaelfitzhavey) [inline orchestration-ui/src/containers/Lights/components/Light.vue]: let's remove this if we''re not using it anywhere?

File: orchestration-ui/src/containers/Lights/components/Light.vue [hunk 3]
```diff
@@ -39,8 +39,15 @@
...
+					<verte
+						picker="wheel"
+						model="rgb"
+						display="picker"
+						:rgbSliders="false"
+						:showHistory="false"
+						:draggable="false"
+						:enableAlpha="false"
+						v-model="inputVal.state.colour"
 						@input="onColourPickerInput"/>
```
Comment (michaelfitzhavey) [inline orchestration-ui/src/containers/Lights/components/Light.vue]: let's remove this `@input` handler as it's just there for science

File: orchestration-ui/src/containers/Lights/components/Light.vue [hunk 4]
```diff
@@ -109,7 +117,7 @@ export default {
 	},
 	methods: {
 		onColourPickerInput() {
-			this.inputVal.state.colour = this.colourPickerValue.hex;
+			console.log(this.colourPickerValue);
```
Comment (michaelfitzhavey) [inline orchestration-ui/src/containers/Lights/components/Light.vue]: as mentioned above, let's remove this method

**Comment by andrewlew1s** [comment]
This was a fantastic effort

File: orchestration-ui/src/containers/Lights/components/Light.vue (hunk 4 above)
Comment (dan47bennett) [reply orchestration-ui/src/containers/Lights/components/Light.vue]: All this redundant code has now been removed.

File: orchestration-ui/src/containers/Lights/components/Light.vue (hunk 3 above)
Comment (dan47bennett) [reply orchestration-ui/src/containers/Lights/components/Light.vue]: Removed.

File: orchestration-ui/src/containers/Lights/components/Light.vue (hunk 2 above)
Comment (dan47bennett) [reply orchestration-ui/src/containers/Lights/components/Light.vue]: Removed.

**Pull request merged**